OmiseSDK/CardNumber.swift

```diff
@@ -15,7 +15,11 @@ public final class CardNumber {
             .first
     }
     
-    public static func format(pan: String) -> String {
+    @objc(brandForPan:) public static func __brand(pan: String) -> Int {
```

Comment:
Anyone has a better idea than this? I have 2 ideas at first,
1. Return the raw value of the CardNumber instance 
2. Declare a new case on CardNumber represents the invalid card.
I choose no. 1 for now